src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user: { name, email } }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${email}`}>
+      {name}
+    </a>
```

Comment:
Add condition for rendering if use is not undefined
```suggestion
   <>
     {user && (
        <a className="UserInfo" href={`mailto:${email}`}>
          {name}
        </a>
     )}
   <>
```